framework/src/org/checkerframework/common/basetype/messages.properties

```diff
@@ -18,6 +18,8 @@ bound.type.incompatible=incompatible bounds in %s\ntype: %s\nupper bound: %s\nlo
 monotonic.type.incompatible=cannot assign %s to %s (monotonic type).\nfull type found: %s
 type.invalid=invalid type: annotations %s in type "%s"
 type.invalid.conflicting.annos=invalid type: conflicting annotations %s in type "%s"
+type.invalid.too.many.annotations=invalid type: too many annotations %s in type "%s"
```

Comment:
This key doesn't seem to be used. Remove it? Or document that it's for future extension?